earth_enterprise/settings.gradle

```diff
@@ -0,0 +1 @@
+
```

Comment:
If we add the file here we should add an include line to point to the RPM folder so that becomes the child folder.